frontend/test/metabase/scenarios/custom-column/reproductions/25189-cc-column-reference-only.cy.spec.js

```diff
@@ -0,0 +1,75 @@
+import { restore, filter, summarize } from "__support__/e2e/helpers";
+import { SAMPLE_DATABASE } from "__support__/e2e/cypress_sample_database";
+
+const { ORDERS, ORDERS_ID } = SAMPLE_DATABASE;
+
+const ccTable = "Custom Created";
+const ccFunction = "Custom Total";
+
+const questionDetails = {
+  name: "25189",
+  query: {
+    "source-table": ORDERS_ID,
+    limit: 5,
+    expressions: {
+      [ccTable]: ["field", ORDERS.CREATED_AT, null],
```

Comment:
This is the only thing that really matters for this reproduction, because CC reference is a single base-column only.

But I've added a check for a custom column based on a custom expression (`case` function) as an additional check.